spec/features/visitor_can_increase_and_decrease_quantities_in_cart_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe "Visitor has an item in cart" do
+  it "and increases the quantity in cart" do
+    category = Category.create(name: "scuba")
```

Comment:
this is what I did for categories in my test. But before i PR my test, we should regroup in the AM to make sure we're all on the same syntax at least to lower the risk of a lot of merge conflicts.